plugins/modules/efs.py

```diff
@@ -274,6 +280,7 @@ class EFSConnection(object):
     STATE_AVAILABLE = 'available'
     STATE_DELETING = 'deleting'
     STATE_DELETED = 'deleted'
+    BACKUP_DISABLED = 'DISABLED'
```

Comment:
I'm not sure the constant increase the readability.

Reply:
would `BACKUP_POLICY_DISABLED` or `BACKUP_POLICY_STATUS_DISABLED`  improve on readability?